library/std/src/backtrace.rs

```diff
@@ -268,7 +268,7 @@ impl Backtrace {
                 Err(_) => false,
             },
         };
-        ENABLED.store(enabled as usize + 1, SeqCst);
+        ENABLED.store(enabled as usize + 1, Relaxed);
```

Comment:
How about storing boolean as is in atomic?
I managed to remove 1 branch and make code shorter by this:
[godbolt](https://godbolt.org/#z:OYLghAFBqd5TKALEBjA9gEwKYFFMCWALugE4A0BIEAZgQDbYB2AhgLbYgDkAjF%2BTXRMiAZVQtGIHgBYBQogFUAztgAKAD24AGfgCsp5eiyahSAVyVFyKxqiIEh1ZpgDC6embZMQAZgBs5M4AMgRM2AByngBG2KQgAQAO6ErEDkxuHl6%2BicmpQiFhkWwxcQE22HZpIkQspEQZnt7%2B1ti29kLVtUQFEdGx8dY1dQ1ZzZZdPUUl8QCU1uhmpKicXBbYANSWmCAgSgCeTKg7LCRsBEcgAKQA7ABCAIKn58oEAF7Y5OsA8qQ4pKHAG4AEUuPluly09wh9wSZii6xoTHWxFiAH0SOgIAQlKjmCwooxtutLgAmPyPdBnVAvd6fJDYFgANz2qIwbFhRE4CKYEBmAFpQbgouh3PzBcL3DdwZD1rL1owiOtxKh6cSfEDkTi8QTsJgAHT0dAsTAQH5/AE7ABKrRY6l1M1B0vucvWbBOKqVLBVGyl0JdLq0xOuLnWPDVIJ8uHWpGwREWSOVqtBEY1PHIfv9stR4cFQfB1xBMrlwOhGdlCuj2CU4fW9KZLLZHOwvMdZc1uNYOv1ljIzZj1ZY1Ysbw%2B31%2BsQtIGtRjtmAdYLb/dLBehXDm9G4AFZ%2BN4uDpyOhuJaLIqlAslj6ST4%2BOQiNo13MANYgaTXPWvrR%2BHxaTckrQADn/a5N1kDcuGkfg2BATctHIXd90PLh%2BCUEBYLvPc13IOBYBQDAcHwYgyEoag6EYVgOG4G9BGEMQJE4GQ5GEZQ1E0DDyH0ElDGMUwTxaNo0icJhXHcRpfGuQIhMmPo4h4NMkhSdp0hE0ZxPkvImCk4p%2BlkviKkUzphmUppxPKSoOiGbpQl6LSZLTcZDMyYzBgmKypm0ng5jPRZlm4NZNiIbZdgOC4Tkpc4dilCkqRpUczQnEwSwXSFoVheFEWRTlSHREUsS1TtCRAYkySi54UlpWsGWZVlKSbQrEV5AVIwlegxSakV6F9ItZTdIgPWxDt8UJA0jRNOL/hMK0bVnB07jbAMc0jPNgXTLr/TDZNcxjONSCRGgJBUFbnUzdZsw2xatvjdYiHMD42xLZLVorfsazrKrGzMTkWySo7ZX67Uhp7GMIGewd1mHd4ituUN1k%2BMbJ2nW17VbVal2SldIUwsDtzg%2B8DyPE9NnPZYip8El%2BHQnQZjmOs/modduAg8goJgnG2MQ5DUNve8qfIZ9X3fa5P2/X8AKAkDDG4Hwd1x9muYwuZsIQeAIFwmqGFiYiIEbdW4lIHhN0FgQGEylCICiXGolCWo9ko/g2Q4YQviYegbbYnA3RMSQ3YIGNKkZKtcewO1UA%2BlYb1CTkwP3egCCiUhrbcHBbdvf4oN4TCaCMYAlAANQIbAAHcvgSZhk%2Bo0RxEkBjy%2BYjRcY4riJvMSxDFjlDIDmdAEkUlCuD5EMu6IPlGH9jr1R8dZkNaPSBIgZwRm8NNglc6SDDUxSF7X3JFM06Y7OnsymAM%2BojIMUz9Is3ftOchzRLsy%2BV5sqRPKJ%2BiU%2BwMPMa3aW2e4ZvetDHqA2epAwQGPC3dYBESCkBJmmdYbh2Q6xJiSGY5NuZPmgiSIBJIACc/gcGvhwbJEk0gAhgUZszWC8F%2BCyxQmhbmWFEAqzQGrRgFAqBa1Yf0PWBtYKkRNtQc2bFLasFIK7G89tmBECdi7XG7tuJe33IQX29h/a933EHCoodk4R1aLjGOccE5YBWPua6BA058DmJnFg2c86F2LqXdOjEK50SkLIGuKg65sX0GmIwTdeIGPbryA83c0i937usQew9sCj2TFLXSh9BLCUcgYZehRV5yW3mkTemSFJpCvjJBJF8ug5KKVUB%2B6Sn73xKafapdQCnP3mN5TgnFrof1tvTLg2NqF4y4P/VAgDgGgPAVYSBhBoHIM%2BAghISDSTXlQXLSm1MGS02CeQyC0EqEy24Bzeh8sMG/mwXgvwBDrhEJ4CQshksf4IR2Ysh8EsuBk1ZrcpC9yeb%2B1ICkRw0ggA%3D)